src/app/material-docs-app.html

```diff
@@ -1,2 +1,2 @@
-<app-navbar [class.mat-elevation-z6]="showShadow"></app-navbar>
+<app-navbar class="mat-elevation-z6"></app-navbar>
```

Comment:
Any way to only make the elevation show once the page has been scrolled and stickiness has taken effect? If not, I think `showShadow` can be refactored out of the component class.

Reply:
I don't believe its worth the effort to make the elevation only when scrolled, especially since we are faking the stickiness of the header.  I have removed `showShadow` as you mentioned.